src/Service/ActivityPub/ApHttpClient.php

```diff
@@ -318,7 +318,7 @@ private function getCollectionObjectImpl(string $apAddress): ?string
             // Accepted status code are 2xx or 410 (used Tombstone types)
             if (!str_starts_with((string) $statusCode, '2') && 410 !== $statusCode) {
                 // Do NOT include the response content in the error message, this will be often a full HTML page
-                throw new InvalidApPostException("Invalid status code while getting: $apAddress, status code: $statusCode");
+                throw new InvalidApGetException("Invalid status code while getting: $apAddress, status code: $statusCode");
```

Comment:
I think you should keep the old exception. There are a lot of places that catch this exception and a few of them count on it. So for the ease of this PR I'd say keep the old one

Reply:
```suggestion
                throw new InvalidApPostException("Invalid status code while getting: $apAddress, status code: $statusCode");
```